src/software/Localization/main_cameraLocalizer.cpp

```diff
@@ -436,7 +436,15 @@ int main(int argc, char** argv)
     {
       POPART_CERR("Unable to localize frame " << frameCounter);
 #if HAVE_ALEMBIC
-      exporter.jumpKeyframe();
+      if(frameCounter == 0)
+      {
+        // No previous pose, use the identity matrix
+        exporter.addCameraKeyframe(geometry::Pose3(), &queryIntrinsics, currentImgName, frameCounter, frameCounter);
+      }
+      else
+      {
+        exporter.jumpKeyframe();
```

Comment:
It's a good idea indeed. We could check value of `mxform.getSchema().getNumSamples()` during the jump. If it is 0, it means that no camera has been registered in the Alembic file, ie we are trying to jump frame 0. In this case we will call `addCameraKeyframe` with default arguments. Or maybe a cleaner way: add an internal function `addDefaultCameraKeyframe()`.